reviewing.md

```diff
@@ -1,5 +1,5 @@
 # How to review conformance results
 
   1. Make sure the list of files matches [the one specified here](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#contents-of-the-pr)
-  2. Look at `version.txt`.  Make sure the `client` and `server` versions match each other to the minor version level.  Make sure this also matches the `vX.Y` subdirectory that the PR is in.
+  2. Look at `version.txt`.  Make sure the `client` version is no more than one minor version greater than the `server` version 
```

Comment:
The guidance here is to make sure that "client" matches the directory they are putting their results into.  So if they are certifying as being "kubernetes 1.7 conformant", they need to be running e2e tests built from 1.7.

Reply:
Sorry I mis-read this now when I re-read it.. I'll swap back.. but the other comment below applies.